app/views/layouts/mailer.html.erb

```diff
@@ -1,13 +1,58 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+    <title>Barong Mailer</title>
```

Comment:
We cannot write barong anywhere
and normally we need to use information from website like application name